### PR TITLE
got the size chart work

### PR DESCRIPTION
**Pull request by vivientam**
i got the size chart work. pls check.

### CODE REVIEW

**Comment by h8rry** [comment]
It says "We can’t automatically merge this pull request.". Can you fix it first? Maybe you need to rebase.

File: app/assets/javascripts/sizecharts.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+// Place all the behaviors and hooks related to the matching controller here.
```
Comment (h8rry) [inline app/assets/javascripts/sizecharts.js]: Can delete this file

File: app/assets/stylesheets/sizecharts.css.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the sizecharts controller here.
```
Comment (h8rry) [inline app/assets/stylesheets/sizecharts.css.scss]: Can delete this file

**Comment by h8rry** [comment]
The rest looks good

File: app/helpers/sizecharts_helper.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+module SizechartsHelper
```
Comment (h8rry) [inline app/helpers/sizecharts_helper.rb]: Can delete

File: app/views/static_pages/sizechart.html.erb [hunk 4]
```diff
@@ -0,0 +1,114 @@
+<h4>Size Chart</h4>
+
+<table class="table table-hover">
+    <thead>
```
Comment (h8rry) [inline app/views/static_pages/sizechart.html.erb:4]: Please use Tab Width = 2 for indentations

**Comment by vivientam** [comment]
changed!

**Comment by h8rry** [comment]
It says "We can’t automatically merge this pull request.". Can you fix it first? Maybe you need to rebase.

**Comment by vivientam** [comment]
I tried to rebase already ... but it seems does work.  see my capture pic from iterm in Colorful Blind git hub group.

**Pull request merged**